.github/workflows/extra.yml

```diff
@@ -118,10 +119,16 @@ jobs:
           echo $IMAGE_ID > image-id.txt
           echo $IMAGE_NAME > image-name.txt
 
-      - name: Make image usable for further builds
+      - name: Make image usable for further builds (if signature_verified property exists)
```

Comment:
Why do we need this check here, but not for the same in the fatimage workflow below?